docs/compilers/Visual Basic/Compiler Breaking Changes - DotNet 10.md

```diff
@@ -35,3 +35,24 @@ Class C
 End Class
 ```
 
+## `Microsoft.CodeAnalysis.EmbeddedAttribute` is validated on declaration
+
+***Introduced in Visual Studio 2022 version 17.13***
+
+The compiler now validates the shape of `Microsoft.CodeAnalysis.EmbeddedAttribute` when declared in source. Previously, the compiler
+would allow user-defined declarations of this attribute, but only when it didn't need to generate one itself. We now validate that:
```

Comment:
>, but only when it didn't need to generate one itself [](http://example.com/codeflow?start=55&length=53)

", but only when it didn't need to generate one itself" Is this part accurate for VB?

Reply:
Technically, yes, in that it never generated one, so it never forbid a user definition, but I'll update the wording.